src/components/Recipes.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from "react";
+import { Link, useRouteMatch } from "react-router-dom";
+import {
+  Title,
+  Container,
+  List,
+  ListItem,
+  Text,
+  ItemTitle,
+  Button,
+} from "./Styles.js";
+import api from "../data/DataAPI.js";
+
+function RecipeList({ recipes }) {
+  let match = useRouteMatch();
+  return (
+    <div>
+      {recipes ? (
+        <List>
+          {recipes.map((recipe) => (
+            <ListItem key={recipe.id}>
+              <Link
+                to={`${match.url}/${recipe.id}`}
```

Comment:
cannot get why do you need to use `useRouteMatch`, why not use a relative url here? 

` to={`recipes/${recipe.id}`}`

Reply:
Same here, probably from one of the examples of React Router. This is not needed indeed.